customer-churn/streamlit_apps/streamlit_app_kubeflow.py

```diff
@@ -0,0 +1,101 @@
+import streamlit as st
+from zenml.repository import Repository
+
+
+def main():
+    st.title("Predicting whether the customer will churn or not before they even did it")
```

Comment:
Change title depending on how we change the zenfile title.

Also, maybe for the streamlit app we can go with something a bit shorter + snappier?

Reply:
I didn't understood the last question tbh

Reply:
Last question was just saying that for streamlit I think it would be nicer for the title to be shorter.

Reply:
Sure, Got it, I will try to make the title short.